tox.ini

```diff
@@ -11,6 +11,7 @@ deps =
     portend
     pytest-services
     munch
+    pathlib
```

Comment:
Why is `pathlib` part of `deps`? I wouldn't expect it to be necessary, since Twine only supports Python 3. Also, I believe this will install https://pypi.org/project/pathlib/, which says:

> Attention: this backport module isn’t maintained anymore.